.golangci.yml

```diff
@@ -281,4 +281,9 @@ issues:
         - gochecknoinits
       # Only safe way to opt-in to protocompile editions support. Temporary and
       # will be removed once opt-in is not necessary.
-      path: private/bufpkg/bufimage/build_image.go
\ No newline at end of file
+      path: private/bufpkg/bufimage/build_image.go
+    - linters:
+        - gochecknoinits
+      # must use init to safely enable editions support for testing
+      # (temporary)
```

Comment:
Can you add a TODO here in the comments to remove this in the future

Reply:
done